internal_workflow_testsuite.go

```diff
@@ -326,11 +327,27 @@ func (env *testWorkflowEnvironmentImpl) executeWorkflowInternal(workflowType str
 	env.startMainLoop()
 }
 
+func (env *testWorkflowEnvironmentImpl) getWorkflowDefinition(wt WorkflowType) (workflowDefinition, error) {
+	hostEnv := env.getHostEnv()
```

Comment:
please remove this env.getHostEnv() and use the global version of getHostEnvironment().